### ISSUE TITLE
translate button does not reappear on map after navigating through screen reader instructions

### ISSUE THREAD

**Issue by cmenedes** [timkeane/nyc-lib#35]
# Current Behavior
Translate button does not reappear on map after navigating through screen reader instructions.
![image](https://user-images.githubusercontent.com/11576888/103552965-bbe11880-4e7a-11eb-9930-8cb22ecafe4a.png)

# Expected Behavior
The user should be able to see the translate button reappear once the screen reader instructions screen is closed. 
![image](https://user-images.githubusercontent.com/11576888/103553124-01054a80-4e7b-11eb-87e7-def6290de0c3.png)
![image](https://user-images.githubusercontent.com/11576888/103553154-0d89a300-4e7b-11eb-93ca-4484dccd35ff.png)
![image](https://user-images.githubusercontent.com/11576888/103553247-314ce900-4e7b-11eb-9265-409996306285.png)

# Failure Information (for bugs)

## Steps to Reproduce
1. Open finderapp.html located in the examples/ol folder
2. Provide splashOptions to the FinderApp constructor: `splashOptions:` {message: 'splash page message'}`
3. Build nyc-lib as described in readme, navigate to dist/examples/ol/finderapp.html
4. In the splash screen, click on 'Screen Reader Instructions' button and then the 'Return To The Map' button. Observe if translate button reappears.

**Comment by cmenedes** [comment]
Addressed in c92567b
